container/kvm/kvm.go

```diff
@@ -184,7 +184,9 @@ func (manager *containerManager) CreateContainer(
 	logger.Tracef("create the container, constraints: %v", cons)
 	if err := kvmContainer.Start(startParams); err != nil {
 		err = errors.Annotate(err, "kvm container creation failed")
-		logger.Infof(err.Error())
+		// Logged as debug here because it is logged as an error in
+		// worker/provisioner.
+		logger.Debugf(err.Error())
```

Comment:
There's no point in logging error messages that we return (especially when they're
less informative than the error that we're returning).

The appropriate place for logging errors is if we're discarding them.